test/test.config.js

```diff
@@ -269,4 +269,16 @@ describe('Config API', function() {
         expect(configResult.toString()).to.be(error);
         done();
     });
+
+    it('should throw error on config.storeName not string', function(done) {
+        let configResult = localforage.config({
+            description: 'The offline datastore for my cool app',
+            driver: localforage.driver(),
+            name: 'My Cool App',
+            storeName: 123,
+            version: 2.0
+        });
+        expect(configResult).to.be(new Error('storeName must be a string'));
+        done();
```

Comment:
```suggestion
```

Reply:
Docs also need to be updated for the same right in the readme. Right??

Reply:
Sorry, my branch name is not following the remaining branches convention of writing using - instead of _. I will take care next time.

Reply:
No worries! I think the docs are okay, `storeName` says:

> The name of the datastore. In IndexedDB this is the dataStore, in WebSQL this is the name of the key/value table in the database. Must be alphanumeric, with underscores. Any non-alphanumeric characters will be converted to underscores.

If you wanted to add that it must be a `String` though, that's fine 🙂

Reply:
Does this branch is ready to merge into master

Reply:
Why there is no dev branch in this project and only master + feature branches. Won't it be complex to handle the project in such a way??

Reply:
Nope, this approach has worked just fine for us. 🙂

I'm not a big fan of `develop` or `dev` branches personally, and for this project it would be an extra maintenance burden.

Reply:
> Does this branch is ready to merge into master?

??

Reply:
> Nope, this approach has worked just fine for us. 🙂
> 
> I'm not a big fan of `develop` or `dev` branches personally, and for this project it would be an extra maintenance burden.

Ok. Cool.

Reply:
> Does this branch is ready to merge into master?

We merge PRs into the `master` branch, yes.